championship_manager.go

```diff
@@ -966,9 +993,6 @@ func (cm *ChampionshipManager) ImportChampionship(jsonData string) (string, erro
 		return "", err
 	}
 
-	// generate a new ID to avoid clashes
```

Comment:
Has this changed just so imported championships will replace existing ones with the same ID?